### PR TITLE
[VEN-2869] Update Prime Distributions and Allocations

### PR DESCRIPTION
**Pull request by web3rover**
## Description

Resolves VEN

### CODE REVIEW

**Review by GitGuru7 — APPROVED** [review]

**Review by chechu — COMMENTED** [review]
Commands lgtm

**Review by chechu — APPROVED** [review]

File: vips/vip-383/bscmainnet.ts [hunk 1]
```diff
@@ -0,0 +1,61 @@
+import { ProposalType } from "src/types";
+import { makeProposal } from "src/utils";
+import { FAST_TRACK_TIMELOCK } from "src/vip-framework";
+
+const vip383 = () => {
+  const meta = {
+    version: "v2",
+    title: "VIP-383: [Ethereum] Prime Adjustment Proposal - Q4 2024",
```
Comment (kkirka) [inline vips/vip-383/bscmainnet.ts:8]: I think we don't use colums in VIP titles usually, the only occurrence is VIP-381



```suggestion
    title: "VIP-383 [Ethereum] Prime Adjustment Proposal - Q4 2024",
```

Comment (chechu) [reply vips/vip-383/bscmainnet.ts]: https://github.com/VenusProtocol/vips/pull/408/commits/80094dc9ffc2f97103f560564c5bdc88f4626c55

**Review by chechu — APPROVED** [review]

**Comment by chechu** [comment]
https://app.venus.io/#/governance/proposal/383?chainId=56

**Pull request merged**